### ISSUE TITLE
Add manual shift-a harvesting tools

### ISSUE THREAD

**Issue by Unrepentant-Atheist** [Noctifer-de-Mortem/nocs_cata_mod#33]
Never sure which mod to post suggestions to, so here goes too: https://github.com/pisskop/PKs_Rebalancing/issues/22

**Comment by Noctifer-de-Mortem** [comment]
When you mean fast harvesting, you mean being able to harvest a plant in seconds?

**Comment by Unrepentant-Atheist** [comment]
Yes, just like tilling soil by simply pressing shift-a with the hoe. Harvesting doesn'T have to include putting stuff in players inventory. Just shift-a, plant harvested, possibly with some sort of bonus if tool can have +harvesting on it like other tools have butchering, digging, etc, if not, that's fine too.

**Comment by Noctifer-de-Mortem** [comment]
What you are describing might require having to add a new (a)ction and I am not sure if that require messing with the game's source code. If it doesn't or there is no need then a simple override to the scythe could do it.

**Comment by Unrepentant-Atheist** [comment]
I hope it's possible, if not, maybe do a pull request with the new game code? I dunno, I don't know the game's code and how the devs do pulls.

**Comment by pisskop** [comment]
I think it does require hardcoding a new action.  Its possible, but you'll have to appeal to people who know more about the code.  Farming is going to be improved as npcs are, for sure

**Comment by Noctifer-de-Mortem** [comment]
I will leave this open to to see if it can be done eventually. But it will require messing with the core code (not skilled enough).
